onnxruntime/core/graph/contrib_ops/diffusion_defs.cc

```diff
@@ -44,9 +44,13 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
         .Attr("activation",
               "Activation after group normalization: 0 for None, 1 for Swish",
               AttributeProto::INT)
+        .Attr("channels_last",
```

Comment:
Need update operator documents (You can find updated documents from the artifact of `Windows GPU CI Pipeline (kernelDocumentation build_x64_RelWithDebInfo)`) of this pull request.